docs/astro/src/content/docs/guide/language/coding/file.mdx

```diff
@@ -338,9 +338,9 @@ The path to each library, as file or directory, must be defined separately at co
 Use one of the following methods to help the Slint compiler resolve libraries to the correct
 path on disk:
 
-* When using Rust and `build.rs`, call [`with_library_paths`](slint-build-rust:struct.CompilerConfiguration#method.with_library_paths)
+* When using Rust and `build.rs`, call [`with_library_paths`](https://docs.slint.dev/latest/docs/rust/slint_build/struct.CompilerConfiguration#method.with_library_paths)
```

Comment:
Hmmm, when we link from the Rust API docs to the Slint docs, we use a versioned link, so releases.slint.dev/1.9.1/docs/rust/index.html does link to https://releases.slint.dev/1.9.0/docs/slint/ . Shouldn't the links back also be versioned then?

(This is not a blocker for this PR, as the previous link was broken altogether, more a general comment/thought :-)